qemu/tests/block_check_max_tranfer_length.py

```diff
@@ -0,0 +1,44 @@
+from virttest import error_context
+from virttest import utils_disk
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check the maximum transfer length:
+
+    1) Start guest with a data disk.
+       Combined of seg_max and queue_depth.
+    2) Do format disk in guest.
+    3) Check the maximum transfer length.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    img_size = params.get("image_size_stg", "10G")
+    expect_max_transfer_length = params["expect_max_transfer_length"]
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+
+    check_cmd = utils_misc.set_winutils_letter(session,
+                                               params["check_cmd"])
+    error_context.context("Format data disk", test.log.info)
+    disk_index = utils_disk.get_windows_disks_index(session, img_size)[0]
+    if not disk_index:
+        test.error("Failed to get the disk index of size %s" % img_size)
```

Comment:
```suggestion
    disk_index = utils_disk.get_windows_disks_index(session, img_size)
    if not disk_index:
        test.error("Failed to get the disk index of size %s" % img_size)
     disk_index = disk_index[0]
```
The function `get_windows_disks_index` will return an empty list if no such disks, so then getting the index 0 of it will raise `IndexError: list index out of range`.  
The `test.error("Failed to get the disk index of size %s" % img_size)` will never be called.